src/utils/BlockUtils.e2e.ts

```diff
@@ -0,0 +1,34 @@
+import { providers } from "ethers";
+import { averageBlockTime } from "./BlockUtils";
+
+type Provider = providers.Provider;
+
+const networks: { [chainId: number]: string } = {
```

Comment:
For future, I will propose to migrate the "node url selection" logic we have in relayer-v2 over to sdk-v2, so it can be used in all of our e2e tests (and provide sensible public RPCs if private ones are not defined).

Reply:
definitely +1